odxtools/diagdatadictionaryspec.py

```diff
@@ -115,25 +122,13 @@ def from_et(et_element: ElementTree.Element,
         else:
             unit_spec = None
 
-        # TODO: Parse different specs.. Which of them are needed?
-        for (path, name) in [
-            ("STATIC-FIELDS", "static fields"),
-            ("DYNAMIC-LENGTH-FIELDS/DYNAMIC-LENGTH-FIELD", "dynamic length fields"),
-            (
-                "DYNAMIC-ENDMARKER-FIELDS/DYNAMIC-ENDMARKER-FIELD",
```

Comment:
`DYNAMIC-ENDMARKER-FIELD` is now supported?

Reply:
nope, but it will be soon and this message will be not seen by too many people?!